pkg/action/build-uki.go

```diff
@@ -322,8 +322,11 @@ func (b *BuildUKIAction) createInitramfs(sourceDir, artifactsTempDir string) err
 			return fmt.Errorf("getting record of %q failed: %w", filePath, err)
 		}
 
-		rec.Name = strings.TrimPrefix(rec.Name, sourceDir)
-		if err := rw.WriteRecord(rec); err != nil {
+		if rec.Name != strings.TrimPrefix(rec.Name, sourceDir) {
+			rec.Name = strings.TrimPrefix(rec.Name, sourceDir)
```

Comment:
Too late (merged), but still, if this isn't a costly operation, it doesn't need to be in an `if`. Just set:

```
rec.Name = strings.TrimPrefix(rec.Name, sourceDir)
```
always, like it was before.

Reply:
Thing is, I have no idea why this is here and I could not get it to trigger, so it may not be needed at all?

So to preserve the filenames as they are in the source I rather do it only when it's needed

Reply:
but if `rec.Name` is already equal to the `TrimPrefix` result, re-assigning the value should have no effect, no? Am I missing something?

Reply:
Nono, you are correct, I'm just missing more coffee today!

This should be fixed as part of the ukifier PR!